frontend/app/src/store/defi/protocols/index.ts

```diff
@@ -266,7 +268,7 @@ export const useDefiSupportedProtocolsStore = defineStore(
             for (const asset of assets) {
               const symbol = get(assetInfo(asset))?.symbol ?? asset;
               loans.push({
-                identifier: `${symbol} - ${truncateAddress(address, 6)}`,
+                identifier: `${symbol} - ${address}`,
```

Comment:
Ok, so the question is do we use the identifier here for anything other than display?

Re-parsing everything is a bit silly

Reply:
Ok from what I can see the information seems to be already there.